addons/crm/report/crm_opportunity_report_views.xml

```diff
@@ -61,11 +61,11 @@
             <field name="priority">32</field>
             <field name="arch" type="xml">
                 <search string="Opportunities Analysis">
-                    <filter string="My Opportunities" name="my"
+                    <filter string="My Pipeline" name="my"
                             domain="[('user_id', '=', uid)]"/>
                     <separator/>
-                    <filter name="lead" string="Lead" domain="[('type','=', 'lead')]" help="Show only lead"/>
-                    <filter name="opportunity" string="Opportunity" domain="[('type','=','opportunity')]" help="Show only opportunity"/>
+                    <filter name="opportunity" string="Opportunities" domain="[('type','=','opportunity')]" help="Show only opportunity" groups="crm.group_use_lead"/>
```

Comment:
Maybe "show only opportunities", as we probably have more than 1 ?

Reply:
Note that we could also rename the filter to match guidelines (filter_opportunity)